data.sql

```diff
@@ -91,6 +91,13 @@ insert into visits values (2,12,'aug 3, 2020');
 insert into visits values (3,13,'may 24, 2020');
 insert into visits values (1,13,'jan 11, 2021');
 
+/* Advanced DB Milestone_1*/ 
+
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
 
 
```

Comment:
- [ ] This is a pair-programming project means all your team members should contribute in this project. 🤔 But i cannot see any commit from your partner side 👀 Kindly ask your partner to make contributions in this project by adding his commit and resubmit this pull request from your dashboard again for a review 👍